### DISCUSSION TITLE
Add custom context to Slack Notifications

### DISCUSSION THREAD

**Discussion by Arkhas** [spatie/flareapp.io-roadmap#82] [Ideas]
It would be nice to be able to customize the slack notification, or at least add some custom context in it, like Sentry does : 

![image](https://github.com/spatie/flareapp.io-roadmap/assets/5020806/8b44f5cb-fed7-4e92-8d72-d108f4b7ef7f)

Something like that : 

![image](https://github.com/spatie/flareapp.io-roadmap/assets/5020806/616a392e-cf5c-4234-82f4-69c56373015f)

Or if it would be able to implement something like this : 

![image](https://github.com/spatie/flareapp.io-roadmap/assets/5020806/f0bd6384-d38b-4271-af0c-51fa359bb20c)

like in this package : 

https://github.com/spatie/interactive-slack-notification-channel

**Comment by freekmurze** [comment]
Nice idea, added this to our backlog 👍
